app/globals.css

```
@@ -1245,42 +1245,91 @@
+  padding: 8px 12px;
+  border-left: 4px solid hsl(var(--primary));
+  width: auto;
+  max-width: 300px;
+}
+
+.str-chat .str-chat__quoted-message-preview-content {
+  font-size: 13px;
+  color: hsl(var(--foreground));
+  text-align: left;
+  white-space: normal;
+  word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)